furnace/pid1.py

```diff
@@ -136,8 +138,9 @@ def create_default_dev_nodes(self):
 
     def create_loop_devices(self):
         self.create_device_node('loop-control', 10, 237, 0o660)
-        for i in range(8):
-            self.create_device_node('loop{}'.format(i), 7, i, 0o660, is_block_device=True)
+        for loop in self.loop_devices:
+            self.create_device_node(
+                loop.name, 7, loop.minor, 0o660, is_block_device=True)
```

Comment:
Interesting formatting. Is this pep8-compatible?

Reply:
Yes.

Reply:
Wow, that's disappointing. But ok.